plugin/trino-ignite/src/main/java/io/trino/plugin/ignite/IgniteMetadata.java

```diff
@@ -123,6 +130,25 @@ public Optional<ConnectorOutputMetadata> finishInsert(
         return Optional.empty();
     }
 
+    @Override
+    public Optional<ConnectorTableHandle> applyUpdate(ConnectorSession session, ConnectorTableHandle handle, Map<ColumnHandle, Constant> assignments)
+    {
+        // Ignite support row level update, so we should reject this path, earlier than in JDBC client
```

Comment:
Why? If possible we should use `applyUpdate` as this could way more faster. What am I missing?